file.go

```diff
@@ -26,18 +26,18 @@ type File interface {
 
 	addService(s Service)
 
-	lookupComments(name string) string
+	lookupNodeMeta(name string) *NodeMeta
 }
 
 type file struct {
-	desc        *generator.FileDescriptor
-	pkg         Package
-	outputPath  FilePath
-	enums       []Enum
-	msgs        []Message
-	srvs        []Service
-	buildTarget bool
-	comments    map[string]string
+	desc           *generator.FileDescriptor
+	pkg            Package
+	outputPath     FilePath
+	enums          []Enum
+	msgs           []Message
+	srvs           []Service
+	buildTarget    bool
+	nameToNodeMeta map[string]*NodeMeta
```

Comment:
Cosmetic change, I will put `nodeMetas` following the same naming pattern used by the gatherer:

```
	entities map[string]Entity
	pkgs     map[string]Package
	targets  map[string]Package
```

Reply:
Is this really necessary, including for this PR? The single map matches the same logic you already had, in effect.